src/test/java/de/tum/in/www1/artemis/QuizExerciseIntegrationTest.java

```diff
@@ -335,6 +602,26 @@ public void testGetQuizExercise_asStudent() throws Exception {
         checkQuizExerciseForStudent(quizExerciseForStudent_Finished);
     }
 
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void testGetExamQuizExercise() throws Exception {
+        quizExercise = createQuizOnServerForExam(ZonedDateTime.now().plusHours(5), null);
+
+        QuizExercise quizExerciseGet = request.get("/api/quiz-exercises/" + quizExercise.getId(), HttpStatus.OK, QuizExercise.class);
+        checkQuizExercises(quizExercise, quizExerciseGet);
+
+        assertThat(quizExerciseGet).isEqualTo(quizExercise);
+        assertThat(quizExerciseGet).as("Quiz exercise was retrieved").isNotNull();
+        assertThat(quizExerciseGet.getId()).as("Quiz exercise with the right id was retrieved").isEqualTo(quizExerciseGet.getId());
+    }
+
+    @Test
+    @WithMockUser(value = "tutor1", roles = "TA")
+    public void testGetExamQuizExercise_asTutor_forbidden() throws Exception {
+        quizExercise = createQuizOnServerForExam(ZonedDateTime.now().plusHours(5), null);
+        request.get("/api/quiz-exercises/" + quizExercise.getId(), HttpStatus.FORBIDDEN, QuizExercise.class);
+    }
```

Comment:
I don't understand why this test fails: `Response status expected:<201> but was:<403>` 

Does anyone have an idea?

Reply:
Hi Jonas! I looked at your code. The problem is `createQuizOnServerForExam`. This creates the quiz using a POST, but a tutor has not the necessary rights to do this. In order to test this functionality, you should create the quiz exercise without a REST call in the database before trying to get it.

Reply:
Thanks for spotting this!